.vscode/settings.json

```diff
@@ -0,0 +1,140 @@
+{
+	"eslint.validate": [
+		"javascript",
+		{ "language": "typescript", "autoFix": true }
+    ],
+    "eslint.lintTask.options": ". --ignore-path .gitignore",
+    "eslint.lintTask.enable": true,
+    "eslint.workingDirectories": [
```

Comment:
I would suggest adding a script to update this list. I have something similar in a private repo, I will share a gist with my script that parses the output of `lerna ls` to populate this list.

Reply:
https://gist.github.com/jogold/c485409cd9c931633d68504e3e91cee0